phpstan.neon.dist

```diff
@@ -0,0 +1,7 @@
+parameters:
+  bootstrapFiles:
+    - tests/bootstrap.php
+  paths:
+    - src
+    - tests
+  level: 0
```

Comment:
This project is small enough to try and experiment with higher levels, if you feel like trying… raising the bar here shouldn't be too much of a nightmare.